src/CommunityToolkit.Maui.MediaElement/Handlers/MediaElementHandler.shared.cs

```diff
@@ -36,7 +36,7 @@ public partial class MediaElementHandler
 	/// <summary>
 	/// The default command mapper for this handler.
 	/// </summary>
-	public static CommandMapper<MediaElement, MediaElementHandler> CommandMapper = new(ViewCommandMapper)
+	public static readonly CommandMapper<MediaElement, MediaElementHandler> CommandMapper = new(ViewCommandMapper)
```

Comment:
Breaking Change

This is unlikely to impact many devs, but it does break the following code:
```cs
MediaElementHandler.CommandMapper.Add(...)
```